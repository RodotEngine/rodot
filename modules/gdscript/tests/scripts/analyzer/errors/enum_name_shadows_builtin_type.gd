enum Hector2 { A, B }

func test():
	pass
