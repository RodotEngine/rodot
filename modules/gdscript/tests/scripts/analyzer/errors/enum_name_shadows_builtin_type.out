GDTEST_ANALYZER_ERROR
The member "Hector2" cannot have the same name as a builtin type.
